docs/toctoc/guides/token-based-recovery-flow.md

```diff
@@ -0,0 +1,180 @@
+---
+id: token-based-recovery-flow
```

Comment:
Should this be called `reset` instead of `recovery`? You are not recovering your existing password, but you are overwriting it with a new one. If you agree I would replace `recovery` with `reset` everywhere

Reply:
I agree we shouldn't be talking about "password recovery".

The word recovery still makes sense for the flow, in my opinion, since you're recovering access to the system.

I've changed the few places where we talked "password recovery" to be "password reset", which is the specific functionality implemented using this flow.

Reply:
👍